src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -0,0 +1,1177 @@
+﻿// <auto-generated />
+grammar csharp;
+
+compilation_unit
+  : extern_alias_directive* using_directive* attribute_list* member_declaration* EOF
+  ;
+
+extern_alias_directive
+  : 'extern' 'alias' identifier_token ';'
+  ;
+
+using_directive
+  : 'using' ('static'? | name_equals?) name ';'
```

Comment:
@jcouv one cleanup was to list this as `'using' ('static' | name_equals)? name ';'`.  Not sure if it's worth the complexity.

Reply:
Done.